core-aam/aria-brailleroledescription_is_empty_or_whitespace_characters-manual.html

```diff
@@ -0,0 +1,71 @@
+<!doctype html>
+<html>
+  <head>
+    <title>aria-brailleroledescription is empty or whitespace characters</title>
+    <meta content="text/html; charset=utf-8" http-equiv="Content-Type"/>
+    <link rel="stylesheet" href="/wai-aria/scripts/manual.css">
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+    <script src="/wai-aria/scripts/ATTAcomm.js"></script>
+    <script>
+    setup({explicit_timeout: true, explicit_done: true });
+
+    var theTest = new ATTAcomm(
+    {
+   "steps" : [
+      {
+         "element" : "test",
+         "test" : {
+            "ATK" : [
+               [
+                  "property",
+                  "objectAttributes",
+                  "doesNotContain",
+                  "brailleroledescription"
+               ]
+            ],
+            "AXAPI" : [
+               [
+                  "property",
+                  "objectAttributes",
+                  "doesNotContain",
+                  "AXBrailleRoleDescription"
+               ]
+            ],
+            "IAccessible2" : [
+               [
+                  "property",
+                  "objectAttributes",
+                  "doesNotContain",
+                  "brailleroledescription"
+               ]
+            ],
+            "UIA" : [
+               [
+                  "property",
+                  "AriaProperties",
+                  "doesNotContain",
+                  "brailleroledescription"
+               ]
+            ]
+         },
+         "title" : "step 1",
+         "type" : "test"
+      }
+   ],
+   "title" : "aria-brailleroledescription=' '"
+}
+
+    ) ;
+    </script>
+  </head>
+  <body>
+  <p>This test examines the ARIA properties for aria-brailleroledescription is empty or white space characters.</p>
+    <button id="test" aria-brailleroledescription=" ">
```

Comment:
There is discussion in the working group to only filter empty strings, not only-whitespace strings, in which case this test will have to be updated.

Reply:
it was decided to be only the empty space!